python/cugraph/cugraph/tests/data_store/test_gnn_feat_storage_wholegraph.py

```diff
@@ -15,6 +15,8 @@
 import numpy as np
 import os
 
+import numba.cuda
```

Comment:
Do we need to update `dependencies.yaml` to also include this new dependency, or is it already included or covered some other way?

Reply:
Ah, nevermind, I see it's already a dependency.

Reply:
Covered by `cudf` https://github.com/rapidsai/cudf/blob/38261f8509245f88bdeab193a1357d9c73d765f0/dependencies.yaml#L692